### PR TITLE
Add @appetize/playwright & appwright integration

### PR DESCRIPTION
**Pull request by dageforce**
Hi @mxschmitt,

> This is a PR to add Appetize's Playwright integration to the Integrations section of README.

This integration allows users to test their mobile or web applications with Playwright on Appetize's Android emulators and iOS simulators.

Thank you and please let me know if you have any questions!

### CODE REVIEW

**Comment by mxschmitt** [comment]
How does this work? It looks like it doesn't use Playwright's browser automation under the hood and you have a wrapper around webdriver/appium? So far we only added tools which explicitly use Playwright's automation rather than just the test-runner as a wrapper.

**Comment by dageforce** [comment]
Hi @mxschmitt ,

Appetize allows you to run mobile apps in the browser, extremely fast and at scale. 
We tried to address two scenarios with our Playwright integration:
- **Testing Native Apps with Playwright:**
Playwright has stated that [native mobile support is not on their roadmap or planned for the near future](https://github.com/microsoft/playwright/issues/23359).
Appetize has built its own platform-independent automation framework specifically for testing native devices. This solution is unique in that it allows for testing on native apps, cross platform apps, mobile browsers and webviews.  - all from a single test-suite. We believe we provide a perfect complement to Playwright, as it allows teams to have one comprehensive test-suite instead of having to combine multiple suites with different paradigms. 

- **Testing Web Apps on Native Browsers:**
Appetize is close to releasing a solution that will leverage Playwright's automation by connecting to browsers on Appetize's mobile devices (starting with Android). This will allow users to run regular Playwright tests for validating web apps and pages directly on native browsers.

All of this is built on the Appetize infrastructure, so you don't need to set up local emulators or simulators to run tests, whether locally or in CI/CD environments.

**Comment by dageforce** [comment]
Hi @mxschmitt,
I have updated the PR to include appwright and tried to update the text to make it clear that both of these are for mobile testing. Couldn't really think of a well-enough defined new category (which didn't feel more confusing or even harder to find) as these really are integrations, but please let me know if you have a better suggestion on where to place them 👍

File: README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@
 - [playwright-bdd](https://github.com/vitalets/playwright-bdd) - BDD testing with Playwright runner and CucumberJS.
 - [Serenity/JS](https://serenity-js.org) - Acceptance testing, reporting, and test integration framework for Playwright, implementing the [Screenplay Pattern](https://serenity-js.org/handbook/design/screenplay-pattern/).
 - [@guidepup/playwright](https://github.com/guidepup/guidepup-playwright) - VoiceOver and NVDA screen reader driver integration for Playwright.
+- [@appetize/playwright](https://docs.appetize.io/testing) – Mobile tests for web or native apps on [Appetize](https://www.appetize.io)'s virtual devices using Playwright.
+- [appwright](https://www.npmjs.com/package/appwright) – Mobile tests using Appium with Playwright.
```
Comment (mxschmitt) [inline README.md:36]:
```suggestion
- [@appetize/playwright](https://docs.appetize.io/testing) – Mobile tests for web or native apps on [Appetize](https://www.appetize.io)'s virtual devices using Playwright Test Runner.
- [appwright](https://www.npmjs.com/package/appwright) – Mobile tests using Appium with Playwright Test Runner.
```

maybe like that?

Comment (dageforce) [reply README.md]: 👍 Updated.

**Pull request merged**